.github/DISCUSSION_TEMPLATE/migrations.yml

```diff
@@ -0,0 +1,44 @@
+title: "(packageName) Request for migration"
+body:
+  - type: markdown
+    attributes:
+      value: We appreciate your interest in migrating a package to the Chocolatey Packages Repository.
+        Before we can start handling your request, there are a few questions we need you to answer.
+
+        Please also verify that you have looked to see if any existing migration requests have opened.
+
+        Please understand that it may take some time before we decide whether to accept or reject any package migration request.
+        A discussion between team repository maintainers will happen before making the decision.
+  - type: textarea
+    id: reason
+    attributes:
+      label: Why should the package be migrated to this Chocolatey Packages Repository?
+      description: Please describe in detail why you believe the package should be migrated to this repository. The reason must include why the package can not be in your source repository.
+    validations:
+      required: true
+  - type: input
+    id: ccr-link
+    attributes:
+      label: What is the link to the package hosted on Chocolatey Community Repository?
+    validations:
+      required: true
+  - type: input
+    id: source-link
+    attributes:
+      label: Where are the sources located?
+      description: Please provide a link to where we can see the package source outside the Chocolatey Community Repository.
```

Comment:
```suggestion
      description: Please provide a link to where we can see the package source outside the Chocolatey Community Repository. This will normally be a GitHub or GitLab URL.
```